packages/react-core/src/components/Toolbar/__tests__/ToolbarToggleGroup.test.tsx

```diff
@@ -124,9 +124,10 @@ describe('Toolbar', () => {
         <ToolbarContent>{items}</ToolbarContent>
       </Toolbar>
     );
-    expect(view.find('#save-button').exists());
-    expect(view.find('#clear-button').exists());
-    expect(view).toMatchSnapshot();
+    // Expecting 2 matches for text because the buttons also exist in hidden expandable ontent for mobile view
+    expect(screen.getAllByText('Save filters').length).toBe(2);
+    expect(screen.getAllByText('Clear all filters').length).toBe(2);
```

Comment:
Nit: RTL prioritizes `getByRole` with the name property (when applicapble) over `getByText` in [their docs](https://testing-library.com/docs/queries/about/#priority).

```suggestion
    expect(screen.getAllByRole('button', { name: 'Save filters' }).length).toBe(2);
    expect(screen.getAllByRole('button', { name: 'Clear all filters' }).length).toBe(2);
```